### ISSUE TITLE
Add something like xterm modifyOtherKeys or iTerm CSI u

### ISSUE THREAD

**Issue by ruohola** [macvim-dev/macvim#1052]
**Is your feature request about something that is currently impossible or hard to do? Please describe the problem.**
Currently we can't remap for example `<TAB>` and `<C-I>` separately, they both always do the same thing.

**Describe the solution you'd like**
Would be nice if these, and other control keys, could be mapped separately.

**Additional context**
 `<TAB>` and `<C-I>` can already be mapped separately in a Vim 8.2 running xterm https://invisible-island.net/xterm/modified-keys.html or in a Vim running in iTerm2 https://iterm2.com/documentation-csiu.html

**Comment by ychin** [comment]
Yeah this would be a good idea. I can look this up but do you know if gVim also supports this on Windows/Linux?

**Comment by ruohola** [comment]
> Yeah this would be a good idea. I can look this up but do you know if gVim also supports this on Windows/Linux?

Just checked on Windows and at least the out-of-the-box gVim 8.2 does not differentiate those keys.

**Comment by zldrobit** [comment]
I have verified that gVim 9.1 on Linux/Windows does differentiate between <C-I> and <Tab> mappings.
